StarEtc/updRDict.csh

```diff
@@ -0,0 +1,35 @@
+#
+# This mmacro is a temmporary  solution. It ust be the part of "cons"
+# All the *pcm fules creared by Cling must be linked to $STARSYS/lib
+# When this will be done this file will be remmoved.
+#V.Perev
+
+
+set STAR_OBJ = ${STAR}/.${STAR_HOST_SYS}/obj
+cd $STAR_OBJ
+set DIRS = (StRoot StarVMC StDb asps pams)
+
+foreach d ($DIRS)
+##  ls $d
+##	pcm's
+  set pcm = (`find $d -name '*Cint_rdict.pcm'`)
+  foreach p ($pcm)
+    set locName = $p
+
+    set gloName = ${locName:t}
+    set gloName = ${STAR_LIB}/${gloName}
+    if (!(-e "$gloName")) then
+      rm -f $gloName
+      ln -s $STAR_OBJ/$locName $gloName
+    endif
+  end
+##  	Libraies
+  set pcm = (`find $d -name '*.so'`)
+  foreach l ($pcm)
+    set o = ${l:t}
+    set o = ${STAR_LIB}/${o}
+    if (-e "$o") continue
+    rm -f $o
+    ln -s ${STAR_OBJ}/$l $o
+  end
+end
```

Comment:
Please see https://github.com/star-bnl/star-sw/pull/121
The main points to take away from #121 are:

- rootcint in ROOT6 (a.k.a rootcling) apparently produces .pcm files without any additional options
- The produced .pcm files are installed in $STAR_LIB via the Install directive